File: College-Sports-WebApp/ClientApp/src/api/models/competition.ts
/* tslint:disable */
/* eslint-disable */
import { Airing } from '../models/airing';
import { Broadcast } from '../models/broadcast';
import { Competitor } from '../models/competitor';
import { Format } from '../models/format';
import { GeoBroadcast } from '../models/geo-broadcast';
import { Note } from '../models/note';
import { Odd } from '../models/odd';
import { Situation } from '../models/situation';
import { Status } from '../models/status';
import { Ticket } from '../models/ticket';
import { Type } from '../models/type';
import { Venue } from '../models/venue';
export interface Competition {
  airings?: Array<Airing> | null;
  attendance?: number;
  broadcasts?: Array<Broadcast> | null;
  competitors?: Array<Competitor> | null;
  conferenceCompetition?: boolean;
  date?: string | null;
  format?: Format;
  geoBroadcasts?: Array<GeoBroadcast> | null;
  id?: number;
  neutralSite?: boolean;
  notes?: Array<Note> | null;
  odds?: Array<Odd> | null;
  playByPlayAvailable?: boolean;
  recent?: boolean;
  situation?: Situation;
  startDate?: string | null;
  status?: Status;
  tickets?: Array<Ticket> | null;
  timeValid?: boolean;
  tournamentId?: number | null;
  type?: Type;
  uid?: string | null;
  venue?: Venue;
}

File: College-Sports-WebApp/Database/Models/Competition.cs
using System.Collections.Generic; 
namespace College_Sports_WebApp.Database.Models{ 

    public class Competition
    {
        public string id { get; set; }
        public string uid { get; set; }
        public string date { get; set; }
        public int attendance { get; set; }
        public Type type { get; set; }
        public bool timeValid { get; set; }
        public bool neutralSite { get; set; }
        public bool conferenceCompetition { get; set; }
        public bool playByPlayAvailable { get; set; }
        public bool recent { get; set; }
        public Venue venue { get; set; }
        public List<Competitor> competitors { get; set; }
        public List<Note> notes { get; set; }
        public Situation situation { get; set; }
        public Status status { get; set; }
        public List<Broadcast> broadcasts { get; set; }
        public Format format { get; set; }
        public string startDate { get; set; }
        public List<GeoBroadcast> geoBroadcasts { get; set; }
        public int? tournamentId { get; set; }
        public List<Airing> airings { get; set; }
        public List<Ticket> tickets { get; set; }
        public List<Odd> odds { get; set; }
    }

}